run/check-links.mjs

```diff
@@ -62,9 +62,10 @@ Options:
                  (By default, only error URLs are displayed)
   -r, --redirects Show ONLY redirected URLs with their redirect targets
 
-URL Source:
-  URLs are obtained automatically from the getFlatUrls() function in
-  '../src/common/installLinks.mjs' which extracts URLs from the project.
+  URL Source:
+  URLs are obtained automatically from getAllExternalLinks() in '../src/common/urlUtils.mjs' which combines:
+  - '../src/common/installLinks.mjs' (installation and binary URLs)
+  - '../src/common/supportLinks.mjs' (support cases and knowledge base URLs)
```

Comment:
There is no need to state where this function is imported from, we have the import on line 19 for that (DRY right 😉)

Also, we should follow the [core OOP principle of Abstraction](https://stackify.com/oop-concept-abstraction/#:~:text=Its%20main%20goal%20is%20to%20handle%20complexity%20by%20hiding%20unnecessary%20details%20from%20the%20user.) and keep it's implementation where it is defined. Keep it simple 😄